platform/android/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
 ### Bug fixes
 - Load a style without an URI or JSON [#15293](https://github.com/mapbox/mapbox-gl-native/pull/15293)
+- Do not try to wake up the RunLoop if a wake is already pending [#15330](https://github.com/mapbox/mapbox-gl-native/pull/15330) <- Fixes offline downloads that could freeze after resuming.
```

Comment:
How about
 > Do not try to wake up the RunLoop if a wake is already pending. Fixes offline downloads that could freeze after resuming. [#15330](https://github.com/mapbox/mapbox-gl-native/pull/15330)

I'm not sure about the :point_left: structure :upside_down_face:

Reply:
Fixed